package/yast2-samba-server.spec

```diff
@@ -27,9 +27,20 @@ Url:            https://github.com/yast/yast-samba-server
 Group:          System/YaST
 License:        GPL-2.0
 # Service.Active
-BuildRequires:	yast2 >= 3.1.53
-BuildRequires:	libsmbclient libsmbclient-devel perl-Crypt-SmbHash perl-X500-DN samba-client yast2-samba-client perl-XML-Writer update-desktop-files yast2-testsuite yast2-perl-bindings yast2-ldap yast2-users
+BuildRequires:  libsmbclient0
+BuildRequires:  libsmbclient-devel
```

Comment:
Won't libsmbclient0 already be pulled in with "BuildRequires:  libsmbclient-devel"? If so, it might be worth dropping the explicit libsmbclient BuildRequires.